src/universal/modules/teamDashboard/components/ProjectColumn/ProjectColumn.js

```diff
@@ -167,6 +167,9 @@ class ProjectColumn extends Component {
         </div>
         <div className={css(styles.columnBody)}>
           <div className={css(styles.columnInner)}>
+            {(projects.length === 0 && label === 'done') &&
```

Comment:
This conforms to spec, but I'm wondering if a message in the done column, which is often empty, is gonna get old. We're kinda conflating an empty state message with a tip & chances are, if there was a "Don't show this to me again" option, I'd probably click it because text is just noisy & takes away from the important text in the kanban. @jordanh 
Possible alternatives:
- Only show this message for 5 mins after a meeting ends
- when they navigate back to the team dash, animate the DONE cards being archived
- rely on the message in the meeting summary
- only do this for the first 5 meetings